data/language/en-GB.txt

```diff
@@ -2419,7 +2419,7 @@ STR_3327    :Starting positions for people not set
 STR_3328    :Can't advance to next editor stage...
 STR_3329    :Park entrance not yet built
 STR_3330    :Park must own some land
-STR_3331    :Path from park entrance to map edge either not complete or too complex - Path must be single-width with as few junctions and corners as possible
+STR_3331    :Path from park entrance to map edge either not complete or too complex - path must be single-width with as few junctions and corners as possible
```

Comment:
This feels like two complete sentences, so I would say that we should replace the - with a period.  Technically you don't need a period to end the other one if you think it looks better.